.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "rules": {
+        "indent": [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        // "quotes": [
```

Comment:
can we get trailing commas? like everywhere is pretty nice actually.
projects at work use lots of commas.
https://eslint.org/docs/rules/comma-dangle

Reply:
"always" or "always-multiline"?

Reply:
always should be good, sorry for the late reply